lighthouse-core/test/results/sample_v2.json

```diff
@@ -870,6 +880,7 @@
             "startTime": 636.6400000115391,
             "endTime": 1213.2910000218544,
             "transferSize": 139,
+            "resourceSize": 0,
```

Comment:
why do these 0 `resourceSize` items have `transferSize` set?

Reply:
> why do these 0 resourceSize items have transferSize set?

and this makes a lot more sense now when 139 isn't the `transferSize` in KB, it's in bytes :)